scene/main/viewport.cpp

```diff
@@ -119,6 +119,12 @@ RID ViewportTexture::get_rid() const {
 	return proxy;
 }
 
+RID ViewportTexture::get_vp_rid() const {
```

Comment:
Other methods accessing `vp` don't put it in the signature, and it's also not clear what it means at all without checking the source code. How about `get_texture_rid` instead?

Reply:
the clearest would be "get_viewport_texture_rid()", what do you prefer?

Reply:
proposed in the next commit